packages/mobile/e2e/src/usecases/ExchangeCelo.js

```diff
@@ -120,8 +120,9 @@ export default ExchangeCelo = () => {
     // TODO Check that transaction appears in home feed
   })
 
+  // Note: Amount fluctuates based on CELO value
   jest.retryTimes(2)
-  it('Then Sell Minimum CELO', async () => {
+  it.skip('Then Sell Minimum CELO', async () => {
```

Comment:
I wonder if this will be fixed with https://github.com/valora-inc/wallet/pull/662
I'll try it.